README.md

```diff
@@ -231,20 +231,9 @@ You can upload them to the correct S3 bucket with the task `rake flow_explorer:u
 
 ### Translations
 
-We use Transifex for translations.
+Transifex is no longer used for translations.
```

Comment:
I think you can just nix the entire “translations” section

Transifex is still used for translations, it’s just not something we ever interact with in the CLI.

Their website is watching our main branch for new changes to en.yml, and makes a PR with the new es.yml whenever the strings have been 100% translated by translators on the Transifex side

Reply:
Done